build/yamls/offload/antrea-crd.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
same question as for sriov-device-plugin-config.yml below. I don't think the file is referenced anywhere.

If we keep it (e.g. for CI), can we add `mellanox` in the directory hierarchy: `build/yamls/offload/mellanox/antrea-crd.yml`

Reply:
did the same here